### PR TITLE
chore: automate API generator

### PR DESCRIPTION
**Pull request by sranka**
This PR updates the `apis` packages so that

- the process of re-generating the code is automated and documented
- the code is regenerated from an updated swagger file from 2022-09-16 (documentation changes only)
- automatically repair generated typescript so that code blocks are indented at the beginning of the line
- enforce that no eslint warnings are in the generated code, such warnings can corrupt generated API documentation
- change doc links to point to InfluxDB 2.4 (was 2.3)

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] A test has been added if appropriate
- [x] `yarn test` completes successfully
- [x] Commit messages are [conventional](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

**Comment by sranka** [comment]
The CI tests fail on wrong formatting code fence backticks in descriptions. The original swagger descriptions are repaired in https://github.com/influxdata/openapi/pull/541 . Still, the generated index.d.ts file indents the descriptions so that the original error is returned back.

File: packages/apis/DEVELOPMENT.md [hunk 1]
```diff
@@ -10,12 +10,15 @@ $ yarn build
 
 ## Re-generate APIs code
 
+```
+yarn regenerate
```
Comment (bednar) [inline packages/apis/DEVELOPMENT.md:14]: Maybe a note about switch to `packages/apis` directory will be useful:

```suggestion
cd packages/apis

yarn regenerate
```

**Review by bednar — APPROVED** [review]
LGTM 🍻

File: packages/apis/DEVELOPMENT.md (hunk 1 above)
Comment (sranka) [reply packages/apis/DEVELOPMENT.md:14]: it is relative to the markdown file

Comment (bednar) [reply packages/apis/DEVELOPMENT.md:14]: Ok, make sense

**Pull request merged**